include/circt/Dialect/RTL/RTLCombinatorial.td

```diff
@@ -319,3 +319,27 @@ def MuxOp : RTLOp<"mux",
     }]>
   ];
 }
+
+def KnownBitWidthType : Type<CPred<[{getBitWidth($_self) != -1}]>,
+  "Type wherein the bitwidth in hardware is known">;
+
+def BitcastOp: RTLOp<"bitcast", [NoSideEffect]> {
+  let summary = [{
+    Cast one value to to another value of the same size and
```

Comment:
Instead of cast and "to to", I'd recommend "*Reinterpret* one value to another...".